src/module/display-home.js

```diff
@@ -0,0 +1,83 @@
+import showPopupComment from './show-popup-comment.js';
+import addLike from './add-like.js';
+import count from './count-items.js';
+import showlikes from './display-likes.js';
+// import displayComments from './display-comments.js';
```

Comment:
- [ ] it would be better if you remove the commented code in order to keep the file clean and pro